lib/Parser/Json.php

```diff
@@ -116,7 +116,7 @@ function($jComp) use ($self) {
         return $this->root->createComponent(
             $jComp[0],
             array_merge($properties, $components),
-            $defaults = false
+            false
```

Comment:
I kind of like the extra `$defaults`. It doesn't do anything here, but it's a way to make a boolean flag self-documenting. Alternatively this could be replaced by a simple PHP comment.